### PR TITLE
Enabled execution graph panning and zooming

### PR DESCRIPTION
**Pull request by kubama**
SVG now fits into the table and doesn't break layout anymore.

obraz

### CODE REVIEW

File: widgets/executions/src/tasksGraph/ExecutionWorkflowGraph.js [hunk 1]
```diff
@@ -71,11 +81,20 @@ export default class ExecutionWorkflowGraph extends React.Component {
...
             return (
-                <div id="graphContainer">
-                    <svg height={this.state.graphResult.height} width={this.state.graphResult.width}>
-                        <GraphNodes graphNodes={this.state.graphResult.children} />
-                        <GraphEdges graphEdges={this.state.graphResult.edges} />
-                    </svg>
+                <div ref={this.wrapper}>
+                    <UncontrolledReactSVGPanZoom
+                        width={this.state.width}
+                        height={Math.min(380, this.state.graphResult.height)}
```
Comment (qooban) [inline widgets/executions/src/tasksGraph/ExecutionWorkflowGraph.js]: Why 380? Can we have a const for that value?

File: widgets/executions/src/tasksGraph/ExecutionWorkflowGraph.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 /**
  * Created by barucoh on 23/1/2019.
  */
+import { UncontrolledReactSVGPanZoom } from 'react-svg-pan-zoom';
```
Comment (qooban) [inline widgets/executions/src/tasksGraph/ExecutionWorkflowGraph.js]: What is the increase of executions widget bundle after adding that?

Comment (kubama) [reply widgets/executions/src/tasksGraph/ExecutionWorkflowGraph.js]: Before:
```
-rwxrwxrwx 1 jakub jakub 22853 Jan 15 14:55 backend.js
-rwxrwxrwx 1 jakub jakub  4173 Jan 15 14:55 backend.js.br
-rwxrwxrwx 1 jakub jakub  4972 Jan 15 14:55 backend.js.gz
-rwxrwxrwx 1 jakub jakub  2466 Jan 15 14:55 README.md
-rwxrwxrwx 1 jakub jakub   724 Jan 15 14:55 widget.css
-rwxrwxrwx 1 jakub jakub 22780 Jan 15 14:54 widget.js
-rwxrwxrwx 1 jakub jakub  5044 Jan 15 14:54 widget.js.br
-rwxrwxrwx 1 jakub jakub  5700 Jan 15 14:54 widget.js.gz
-rwxrwxrwx 1 jakub jakub 15192 Jan 15 14:55 widget.png
```
After:
```
-rwxrwxrwx 1 jakub jakub 22853 Jan 15 14:37 backend.js
-rwxrwxrwx 1 jakub jakub  4173 Jan 15 14:37 backend.js.br
-rwxrwxrwx 1 jakub jakub  4972 Jan 15 14:37 backend.js.gz
-rwxrwxrwx 1 jakub jakub  2466 Jan 15 14:37 README.md
-rwxrwxrwx 1 jakub jakub   724 Jan 15 14:37 widget.css
-rwxrwxrwx 1 jakub jakub 81385 Jan 15 14:36 widget.js
-rwxrwxrwx 1 jakub jakub 17445 Jan 15 14:36 widget.js.br
-rwxrwxrwx 1 jakub jakub 19677 Jan 15 14:36 widget.js.gz
-rwxrwxrwx 1 jakub jakub 15192 Jan 15 14:37 widget.png
```
So the gziped bundle increased 14KB.

File: widgets/executions/src/tasksGraph/ExecutionWorkflowGraph.js (hunk 1 above)
Comment (kubama) [reply widgets/executions/src/tasksGraph/ExecutionWorkflowGraph.js]: It's a maximum graph height at which the graph can still fit inside the widget (with some margin calculated in). I added a constant to make it descriptive.

**Pull request merged**